ci/run_testsuite_and_record.sh

```diff
@@ -28,6 +29,8 @@ group.user_set.add(user)
 ' | docker exec -i mreg python /app/manage.py shell
 
 # run the test suite
-echo "test" | mreg-cli -u test -d example.org --url http://127.0.0.1:8000 --source testsuite --record new_output.json >/dev/null
-diff testsuite-result.json new_output.json
+echo "test" | mreg-cli -u test -d example.org --url http://127.0.0.1:8000 --source testsuite --record new_testsuite_log.json >/dev/null
+
+# show a detailed diff
+python diff.py testsuite-result.json new_testsuite_log.json
```

Comment:
Dear heaven, this is nice. The output is amazing! Thank you!

A few things:

1. I sometimes get the '---' from the diff on the line with the command, like this:

```diff
host sshfp_remove -fingerprint 394875985 bar---
+++
@@ -11,12 +11,17 @@
             "ipaddresses": [
                 {
                     "macaddress": "",
-                    "ipaddress": "10.0.0.14",
+                    "ipaddress": "10.0.0.12",
                     "host": 14
                 },
                 {
-                    "macaddress": "11:22:33:44:55:66",
-                    "ipaddress": "10.0.0.15",
+                    "macaddress": "",
+                    "ipaddress": "2001:db8::11",
+                    "host": 14
+                },
+                {
+                    "macaddress": "11:22:33:44:55:67",
+                    "ipaddress": "2001:db8::12",
                     "host": 14
                 }
             ],
```

2. Should we add a line for each issue regarding which of `+` and `-` that are expected or tested? Or optionally rename them `T` and `E`? Either way, massive improvement and really really helpful.

Reply:
Thanks! I've made some modifications to `diff.py` now:
- Added a header for each command that should make it more apparent what the pluses and minuses mean
- Fixed the line break problem
- Changed which diff algorithm is used, so the output is more human-readable